### PR TITLE
Speed up Summarize tests in CI

### PR DESCRIPTION
**Pull request by fayvor**
[#149](https://github.com/ibm-granite-community/pm/issues/149)
- Truncate text in CI by looking at GRANITE_TEST env var

### PR Checklist

- [ ] **Commits signed**: All commits must be GPG or SSH signed.
- [ ] **DCO Compliance**: Developer Certificate of Origin (DCO) applies to the code, documentation, and any example data provided. Ensure commits are signed off.
- [ ] **Notebook outputs cleared**: Ensure all notebook outputs are cleared.
- [ ] **Automated testing**: Add the recipe to the automated tests.
- [ ] **Test in Google Colab**:
    - [ ] Test that it works in Google Colab (Python 3.10.12).
    - [ ] Colab has its own package set and Python version, so ensure compatibility.
- [ ] **Test locally**:
    - [ ] Ensure the code works in a fresh Python virtual environment (venv).
- [ ] **Flexible LLM platform support**:
    - [ ] The platform should be easily switchable. Use LangChain for now.
    - [ ] Include `!pip install git+https://github.com/ibm-granite-community/granite-kitchen` in the instructions.
- [ ] **Example data**: Follow the example data guidance.
- [ ] **README.md updates**:
    - [ ] Add a link to the recipe in the Table of Contents (ToC).
    - [ ] Include a Colab button after that link.

### CODE REVIEW

**Comment by fayvor** [comment]
Tests will fail until this is merged:
https://github.com/ibm-granite-community/utils/pull/19

File: recipes/Summarize/Summarize.ipynb [hunk 1]
```diff
@@ -239,8 +239,8 @@
     "from langchain.text_splitter import TokenTextSplitter\n",
     "from langchain.docstore.document import Document\n",
     "\n",
-    "excerpt_length = 20000\n",
-    "text = book_contents # [:excerpt_length]\n",
+    "# Use entire book, or truncate if testing.\n",
+    "text = book_contents if not get_env_var('GRANITE_TEST', 'false') == 'true' else book_contents[:20000]\n",
```
Comment (rawkintrevo) [inline recipes/Summarize/Summarize.ipynb:243]: `if get_env_var('GRANITE_TEST').lower() == 'true' book_contents[:20000]

**Review by rawkintrevo — CHANGES_REQUESTED** [review]

File: recipes/Summarize/Summarize.ipynb (hunk 1 above)
Comment (adampingel) [reply recipes/Summarize/Summarize.ipynb:243]: The name of the variable is `GRANITE_TESTING`

Comment (fayvor) [reply recipes/Summarize/Summarize.ipynb:243]: Will update.

Comment (fayvor) [reply recipes/Summarize/Summarize.ipynb]: Agentic_RAG is failing trying to load langchain_ibm for WatsonxLLM.

**Pull request merged**